docs/headless-cms/06-01-docstopic.md

```diff
@@ -49,26 +49,49 @@ status:
-| **status.reason** |    **Not applicable**   | Provides the reason why the DocsTopic CR processing succeeded, is pending, or failed.  |
+| **metadata.name** | Yes | Specifies the name of the CR. It also defines the **cms.kyma-project.io/docs-topic** label added to the Asset CR that the DocsTopic CR defines. Because of label name limitations, DocsTopic CR names can have a maximum length of 63 characters. |
+| **metadata.labels** | No | Specifies how to filter and group Asset CRs that the DocsTopic CR defines. See [this](#details-headless-cms-in-console) document for more details. |
+| **spec.displayname** | Yes | Specifies a human-readable name of the DocsTopic CR. |
+| **spec.description** | Yes | Provides more details on the purpose of the DocsTopic CR. |
+| **spec.sources** | Yes | Defines the type of the asset and the **cms.kyma-project.io/type** label added to the Asset CR.  |
+| **spec.sources.type** | Yes | Specifies the type of assets included in the DocsTopic CR. |
+| **spec.sources.name** | Yes | Defines an identifier of a given asset. It must be unique if there is more than one asset of a given type in a DocsTopic CR. |
+| **spec.sources.mode** | Yes | Specifies if the asset consists of one file or a set of compressed files in the ZIP or TAR format. Use `single` for one file and `package` for a set of files.  |
+| **spec.sources.parameters** | No | Specifies a set of parameters for the asset. For example, use it to define what to render, disable, or modify in the UI. Define it in a valid YAML or JSON format. |
+| **spec.sources.url** | Yes | Specifies the location of a single file or a package. |
+| **spec.sources.filter** | No | Specifies a set of assets from the package to upload. The regex used in the filter must be [RE2](https://golang.org/s/re2syntax)-compliant. |
+| **status.lastheartbeattime** | Not applicable | Provides the last time when the DocsTopic Controller processed the DocsTopic CR. |
+| **status.message** | Not applicable | Describes a human-readable message on the CR processing progress, success, or failure. |
+| **status.phase** | Not applicable | The DocsTopic Controller adds it to the DocsTopic CR. It describes the status of processing the DocsTopic CR by the DocsTopic Controller. It can be `Ready`, `Pending`, or `Failed`. |
+| **status.reason** | Not applicable | Provides the reason why the DocsTopic CR processing succeeded, is pending, or failed. See the [**Reasons**](#status-reasons) section for the full list of possible status reasons and their descriptions. |
 
 > **NOTE:** The DocsTopic Controller automatically adds all parameters marked as **Not applicable** to the DocsTopic CR.
 
+### Status reasons
+
+Processing of a DocsTopic CR can succeed, continue, or fail for one of these reasons:
+
+| Reason | Phase | Description |
+| --------- | ------------- | ----------- |
+| `AssetCreated` | `Ready` | The Asset Controller created the specified asset. |
```

Comment:
Pending, not Asset Controller, CMS - also in other places ;)

Reply:
Changed to DocsTopic Controller here and everywhere else.